sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRDD.scala

```diff
@@ -216,14 +159,14 @@ private[jdbc] class JDBCRDD(
     getConnection: Int => Connection,
     schema: StructType,
     columns: Array[String],
-    filters: Array[Filter],
+    predicates: Array[Predicate],
     partitions: Array[Partition],
     url: String,
     options: JDBCOptions,
-    groupByColumns: Option[Array[String]],
-    sample: Option[TableSampleInfo],
-    limit: Int,
-    sortOrders: Array[SortOrder])
+    groupByColumns: Option[Array[String]] = None,
+    sample: Option[TableSampleInfo] = None,
+    limit: Int = 0,
+    sortOrders: Array[SortOrder] = Array.empty[SortOrder])
```

Comment:
why do we add default parameter values? are they necessary?

Reply:
No need anymore.